cmd/resource-topology-exporter/main.go

```diff
@@ -123,13 +125,17 @@ func parseArgs(args ...string) (ProgArgs, error) {
 
 	flags.BoolVar(&pArgs.RTE.Debug, "debug", false, " Enable debug output.")
 	flags.StringVar(&pArgs.RTE.TopologyManagerPolicy, "topology-manager-policy", defaultTopologyManagerPolicy(), "Explicitly set the topology manager policy instead of reading from the kubelet.")
+	flags.StringVar(&pArgs.RTE.TopologyManagerScope, "topology-manager-scope", defaultTopologyManagerScope(), "Explicitly set the topology manager scope instead of reading from the kubelet.")
 	flags.DurationVar(&pArgs.RTE.SleepInterval, "sleep-interval", 60*time.Second, "Time to sleep between podresources API polls.")
 	flags.StringVar(&pArgs.RTE.KubeletConfigFile, "kubelet-config-file", "/podresources/config.yaml", "Kubelet config file path.")
```

Comment:
It is not related to the current PR's changes, but why do we have the ```KubeletConfigFile``` argument while we can't access it?

Reply:
We can access, provided we also either provide a selinux configuration to do so, or we elevate the privileges high enough. The first is unpractical, the second is dangerous, so we don't.
That said, up until now here we are just mirroring what u/s does, adding extra features on top. This makes the two (u/s and d/s) completely replaceable, being d/s a superset of u/s.

We will need to evaluate in the near future if we want to actively disable u/s features we don't support, breaking the replaceability.